.github/workflows/ci.yaml

```
@@ -154,7 +212,7 @@ jobs:
         working-directory: ${{ github.workspace }}/src/github.com/${{ github.repository }}
       - name: Interdomain tests
         run: |
-          go test -count 1 -timeout 1h -race -v -run Interdomain
+          go test -count 1 -timeout 1h -race -v -tags=interdomain_test
```

See at comment above.

```suggestion
          go test pkg/internal/interdomain -count 1 -timeout 1h -race -v
```